plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -186,7 +186,7 @@ instance (ExMemoryUsage a, ExMemoryUsage b) => ExMemoryUsage (a, b) where
 instance ExMemoryUsage SatInt where
     memoryUsage n = memoryUsage (fromIntegral @SatInt @Int n)
     {-# INLINE memoryUsage #-}
-deriving newtype instance ExMemoryUsage ExMemory
+
```

Comment:
This was what caused the problems.  There are more `ExMemoryUsage` instances nearby that I'm wondering if we should get rid of.  We only really want instances for built-in types.

Reply:
Go for it, if it compiles then I'm happy, if it doesn't then that tells us about some suspicious stuff.

Reply:
I've now also removed the instances for `SatInt` and `Unique`.